save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/TopBar.kt

```diff
@@ -109,10 +117,29 @@ fun topBar() = FC<TopBarProps> { props ->
                     .split(URL_PATH_DELIMITER)
                     .filterNot { it.isBlank() }
                     .apply {
-                        foldIndexed("#") { index: Int, acc: String, pathPart: String ->
-
-                            val currentLink = "$acc/$pathPart"
-
+                        val insideTab = location.pathname.substringBeforeLast("?").let {
+                            if (it.contains(OrganizationMenuBar.regexForUrlClassification)) {
+                                "organization"
+                            } else if (it.contains(ProjectMenuBar.regexForUrlClassification)) {
+                                "project"
+                            } else if (it.contains(BenchmarkCategoryEnum.regexForUrlClassification)) {
+                                "archive"
+                            } else {
+                                null
+                            }
```

Comment:
When might be better for those purposes, but it is only my opinion, yours is fine too